MekHQ/src/mekhq/campaign/storyarc/StoryArc.java

```diff
@@ -0,0 +1,521 @@
+ * features straightforward and avoids creating complexity and bloat. Most questions of "How can I add this feature?"
+ * should be addressable by creating new StoryPoint and/or StoryTrigger classes and should not require modifying the
+ * fundamental architecture of Story Arcs.
+ *
+ * @author Aaron Gullickson (Taharqa)
+ *
+ */
+public class StoryArc {
+
+    private String title;
+    private String details;
+    private String description;
+
+    private Campaign campaign;
+
+    /** What type of story arc **/
+    private StoryLoadingType storyLoadingType;
+
+    /** A UUID for the initial StoryPoint in this track  - can be null **/
+    private UUID startingPointId;
+
+    /** A hash of all possible StoryPoints in this StoryArc, referenced by UUID **/
+    private Map<UUID, StoryPoint> storyPoints;
+
+    /** A hash of possible personalities that the player might interact with in this story arc **/
+    private Map<UUID, Personality> personalities;
+
+    /** a hash of custom string variables that the creator might specify with a string key **/
+    private Map<String, String> customStringVariables;
+
+    /** directory path to the initial campaign data for this StoryArc - can be null **/
+    private String initCampaignPath;
+
+    /** directory path to this story arc **/
+    private String directoryPath;
+
+    /**
+     * A hash map of replacements for tokens in the narrative strings. The text will be searched for passages matching
+     * the String used for the key and will replace this with the String supplied by the value. Tokens in the text should
+     * be surrounded by curly brackets. For example, &#123;commanderName&#125; in the text would be replaced with the full
+     * name of the most senior active commander in the campaign.
+    **/
+    private static Map<String, String> replacementTokens;
+
+    public StoryArc() {
+        storyLoadingType = StoryLoadingType.BOTH;
+        storyPoints =  new LinkedHashMap<>();
+        personalities = new LinkedHashMap<>();
+        customStringVariables = new LinkedHashMap<>();
+    }
+
+    public void setCampaign(Campaign c) { this.campaign = c; }
+
+    public Campaign getCampaign() { return campaign; }
+
+    private void setTitle(String t) { this.title = t; }
+
+    public String getTitle() { return this.title; }
+
+    public String getDetails() { return details; }
+
+    private void setDetails(String d) { this.details = d; }
+
+    public String getDescription() { return this.description; }
+
+    private void setDescription(String d) { this.description = d; }
+
+    private void setStoryLoadingType(StoryLoadingType type) { this.storyLoadingType = type; }
+
+    public StoryLoadingType getStoryLoadingType() { return storyLoadingType; }
+
+    private void setStartingPointId(UUID u) { this.startingPointId = u; }
+
+    private UUID getStartingPointId() { return startingPointId; }
+
+    public void setInitCampaignPath(String s) { this.initCampaignPath =s; }
```

Comment:
These all need to be three-line declarations with the closing brace on its own line.

Reply:
@Sleet01 Thanks for that catch. I will go through the PR and fix any of those I see.

Reply:
Resolved in latest. I looked through all the files and changed all instances of one line setter/getters to three lines.